GooglePlayInstant/LoadingScreen/LoadingBar.cs

```diff
@@ -13,7 +13,9 @@
 // limitations under the License.
 
 using System.Collections;
+#if UNITY_EDITOR
 using UnityEditor;
```

Comment:
What was using this? Does whatever was using this need to be flagged off to or it will fail to compile as Engine code?